### PR TITLE
Impl [UI Model Endpoints ] Add new tab "Alerts"

### PR DESCRIPTION
**Pull request by adi-gini**

### CODE REVIEW

File: src/common/TabsSlider/TabsSlider.js [hunk 1]
```diff
@@ -189,10 +189,13 @@ const TabsSlider = ({
...
                   className={tabClassName}
                   data-tab={tab.id}
-                  to={generateUrlFromRouterPath(`${window.location.pathname?.replace(/^$|([^/]+$)/, tab.id)}${location.search ?? ''}`)}
+                  to={generateUrlFromRouterPath(
+                    `${window.location.pathname?.replace(/^$|([^/]+$)/, tab.id)}${location.search ?? ''}`
+                  )}
                   onClick={() => onSelectTab(tab)}
                   key={tab.id}
                 >
+                  {tab.icon && <div className="tabs-slider__tab_icon">{tab.icon}</div>}
```
Comment (mavdryk) [inline src/common/TabsSlider/TabsSlider.js]: rename to `-icon`. Underscore is used for modifiers (`_active`, `_disabled`, `_selected`, etc.)

File: src/common/TabsSlider/tabsSlider.scss [hunk 2]
```diff
@@ -36,6 +36,17 @@
     &-tip {
       margin-left: 5px;
     }
+
+    &_icon {
+      svg {
+        width: 20px;
+        height: 20px;
+        margin-right: 5px;
+        path {
```
Comment (mavdryk) [inline src/common/TabsSlider/tabsSlider.scss:46]: add empty line before `path`

File: src/components/ModelsPage/ModelEndpoints/modelEndpoints.util.js [hunk 3]
```diff
@@ -60,6 +62,12 @@ const detailsMenu = [
   {
     label: 'metrics',
     id: 'metrics'
+  },
+  {
+    label: 'alerts',
+    id: 'alerts',
+
```
Comment (mavdryk) [inline src/components/ModelsPage/ModelEndpoints/modelEndpoints.util.js]: delete empty line

Comment (mavdryk) [reply src/components/ModelsPage/ModelEndpoints/modelEndpoints.util.js]: the description of the ticket says that there should be a component of the new Alerts tab with a date picker in it. So, this ticket isn't finished

**Review by mavdryk — CHANGES_REQUESTED** [review]

File: src/components/ModelsPage/ModelEndpoints/modelEndpoints.util.js (hunk 3 above)
Comment (ilan7empest) [reply src/components/ModelsPage/ModelEndpoints/modelEndpoints.util.js]: @mavdryk "date picker" will be implemented in separate ticket

File: src/components/DetailsAlerts/DetailsAlerts.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```
Comment (ilan7empest) [inline src/components/DetailsAlerts/DetailsAlerts.js:21]: Add copyrights text above, same as in all .js files

Comment (adi-gini) [reply src/components/DetailsAlerts/DetailsAlerts.js:21]: Done

**Review by mavdryk — APPROVED** [review]

**Pull request closed without merging**